pkg/envoy/eds.go

```diff
@@ -0,0 +1,97 @@
+package envoy
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+
+	envoy_config_endpoint_v3 "github.com/envoyproxy/go-control-plane/envoy/config/endpoint/v3"
+
+	"github.com/openservicemesh/osm-health/pkg/common"
+)
+
+// Verify interface compliance
+var _ common.Runnable = (*HasDestinationEndpointsCheck)(nil)
+
+// HasDestinationEndpointsCheck implements common.Runnable
+type HasDestinationEndpointsCheck struct {
```

Comment:
Can this be renamed into a noun so it doesn't get confused as a function like L85/L92?
Maybe `DestinationEndpointChecker`?